packages/client/wallets/smart-wallet/src/sdk.ts

```diff
@@ -77,18 +70,31 @@ export class SmartWalletSDK {
         return await this.logger.logPerformance("GET_OR_CREATE_WALLET", async () => {
             try {
                 return await this.smartWalletService.getOrCreate(user, chain, walletParams);
-            } catch (error: any) {
+            } catch (error: unknown) {
                 throw this.errorProcessor.map(
                     error,
-                    new SmartWalletError(`Wallet creation failed: ${error.message}.`, stringify(error))
+                    new WalletCreationError(
+                        `Wallet creation failed: ${(error as { message: string }).message}.`,
+                        stringify(error)
+                    )
                 );
             }
         });
     }
 
+    async createPasskeySigner(name: string): Promise<PasskeySigner> {
+        const credential = await WebAuthnP256.createCredential({
```

Comment:
Nice, this is so much faster and better feeling for the user than what we we're doing before, ty!